Moesif.Api/Controllers/BaseController.cs

```diff
@@ -51,7 +51,7 @@ public static IHttpClient ClientInstance
         internal void ValidateResponse(HttpResponse _response, HttpContext _context)
         {
             if ((_response.StatusCode < 200) || (_response.StatusCode > 206)) //[200,206] = HTTP OK
-                throw new APIException(@"HTTP Response Not OK", _context);
+                throw new APIException($"HTTP Response Not OK [status: {_response.StatusCode}, error:{_response.RawBody}]"));
```

Comment:
@XinyuLuan what happened to 2nd parameter `_context` ?

Reply:
Added back